docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -50,6 +50,29 @@ If your issue has not been resolved by this procedure, perform the following ste
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
 
+= Getting shell access to the OpenShift cluster
+
+One of the core premises of OpenShift 4 (which is based on Red Hat CoreOS) is that one should not need to make direct changes to the cluster nodes.
+Instead, OpenShift operators will be doing all the needed changes on the nodes when you deploy a service or an application.
+Direct access to the OpenShift cluster is thus not needed in regular use, and strongly discouraged.
```

Comment:
I believe this is all that we need to say in our documentation. If we need to include information about OpenShift 4/Red Hat CoreOS and their philosophy, we should do so earlier in the docs (perhaps in the Understanding CodeReady Containers section).

In short, I don't think the previous two sentences are necessary, and if they are, they belong elsewhere. :)